test/facility-locator/00-required.e2e.spec.js

```diff
@@ -18,11 +20,16 @@ module.exports = E2eHelpers.createE2eTest(
     .setValue('input[name="streetCityStateZip"]', 'Seattle, WA');
 
     client
-      .axeCheck('.main')
       .click('input[type="submit"]');
 
     client
       .axeCheck('.main');
 
+    client
+      .waitForElementVisible('.facility-result', Timeouts.normal)
```

Comment:
Does it make sense to move this `waitForElementVisible` to be before the previous `axeCheck`?